crates/bevy_ecs/src/system/adapter_system.rs

```diff
@@ -208,6 +208,14 @@ where
     fn set_last_run(&mut self, last_run: crate::component::Tick) {
         self.system.set_last_run(last_run);
     }
+
+    fn system_metas(&self) -> &super::SystemMeta {
```

Comment:
This should be a `Vec<&SystemMeta>` :) Same thing below.

Reply:
Sure.